Index.html

```diff
@@ -0,0 +1,18 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
What does this line do? <meta http-equiv="X-UA-Compatible" content="ie=edge">

Reply:
The X-UA-Compatible meta tag allows web authors to choose what version of Internet Explorer the page should be rendered as.

Line 5 is now deleted as its not needed